consensus/state.go

```diff
@@ -9,6 +9,8 @@ import (
 	"sync"
 	"time"
 
+	errs "github.com/pkg/errors"
```

Comment:
again, this prefix is not common in this repo. sorry for being too picky, but single style reduces cognitive load on developer when reading code

Reply:
You are not too picky. It just seems there is no prefix/alias for this particular package in the project though. There is no other place where this is aliased. In other places we use `lerr` for `"github.com/tendermint/tendermint/lite/errors"` and `stderr` for `"errors"`. 

Would `pkgerr` be more suitable?

Reply:
why not `errors`?

Reply:
Because of https://github.com/tendermint/tendermint/pull/2948/files/df20323cd2f261ec8ec83462c866ce602371741a#diff-80343e80650fe43a4394fc26afd5cb3bR5

Reply:
you can remove standard `errors` package. `pkg/errors` implements standard interface

Reply:
Done, thanks!